lib/api/initiateMultipartUpload.js

```diff
@@ -201,7 +201,7 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
         let uploadId;
         if (config.backends.data === 'multiple') {
             return multipleBackendGateway.createMPU(objectKey, metaHeaders,
-            destinationBucket, websiteRedirectHeader, locConstraint, log,
+            bucketName, websiteRedirectHeader, locConstraint, log,
```

Comment:
why the changes from destBucket to bucketName throughout?  they were called destination bucket since the objects live in some middle-world as mpu's until they are completed.

Reply:
`destBucket` is an object representing the metadata of the bucket.
`bucketName` is the actual name of the bucket.